### ISSUE TITLE
Authenticating always fails

### ISSUE THREAD

**Issue by RenaudBompuis** [jvatic/audible-downloader#11]
Not sure if it's me or if something has changed on Audible's side but I'm systematically getting these errors when using the US site:

```
There was a problem: Enter a valid email or mobile number
Error authenticating: step[6]: auth verification failed
```

**Comment by jvatic** [comment]
Yeah, I think there may be either different markup breaking cross-region support, and/or additional prompts that aren't handled. I'm not sure when I'll have time to dig into this. PRs are always welcome ;)

**Comment by jasonehines** [comment]
I'm getting this on MacOS as well. I was able to authenticate the first time, but every time I run the cli or gui, I get that error. Is there some cache I can delete to make the auth a fresh one?

**Comment by jvatic** [comment]
> Is there some cache I can delete to make the auth a fresh one?

Nothing is being cached by the app across runs. My guess is this has to do with Audible serving an altered flow based on your IP that the app doesn't currently handle.

Running either cli or gui with `LOG_LEVEL=trace` env will record responses to `~/audible-downloader-config/debug` (which will be stripped of all or most potentially sensitive information unless `REDACT_DISABLE=true`). Finding the form the app is getting stuck on would be helpful towards getting this fixed.

**Comment by rasmusbe** [comment]
I had the same problem and my solution was to activate 2FA on the account. Seems to work fine with 2FA, but not without.
